### DISCUSSION TITLE
pellet stove with channeled air - flumenfire 12 -ca

### DISCUSSION THREAD

**Discussion by sanya238** [philibertc/micronova_controller#41] [General]
tool is working with flumenfire with channeled air (2 air channel on backside - they have additional fan). 
In this stove in serial socket connected expansion board - I055
But this board have serial connector too (without 5v)
I connected GND and serial to expansion board, and 5v to main board.
Work excellent, but strange fume temp - 10 and flame power only 38%.

**Comment by sanya238** [comment]
And more - new format for sensor
mqtt:
   sensor:
    - state_topic: "micronova/pong"
      name: "pellet stove"
      qos: 0
      icon:  mdi:power
    - state_topic: "micronova/ambtemp"
      name: "pellet stove ambient temp"
      unit_of_measurement: "ºC"
      icon:    mdi:thermometer
      qos: 0
    - state_topic: "micronova/state"
      name: "pellet stove power state"
      qos: 0
      icon: mdi:fire-alert
    - state_topic: "micronova/fumetemp"
      name: "pellet stove flame temp"
      qos: 0
      unit_of_measurement: "ºC"
      icon:    mdi:thermometer
    - state_topic: "micronova/flamepower"
      name: "Pellet stove flame power"
      qos: 0
      unit_of_measurement: "%"
      icon: mdi:fire

**Comment by philibertc** [comment]
Hello,
I seem to have missed this discussion, I apologize for that.
Is there anything new?
I will update the readme for the new MQTT configuration.

Bests,
philibertc
